Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM lambci/lambda:build-python3.6
-MAINTAINER David Redmin <david.redmin@trio.dhs.gov>
+FROM lambci/lambda:build-python3.8
+LABEL maintainer="mark.feldhousen@trio.dhs.gov"
```

Comment:
@hillaryj - You can keep me as the maintainer, unless you want to take over. 😄

Reply:
Restored in https://github.com/cisagov/assessment-data-import-lambda/pull/6/commits/6b43a8120cda6bd7a4b7286e04e729407ec238c0